src/core/qgsapplication.h

```diff
@@ -465,6 +471,7 @@ class CORE_EXPORT QgsApplication : public QApplication
     QgsActionScopeRegistry* mActionScopeRegistry;
     QgsRuntimeProfiler* mProfiler;
     QgsTaskManager* mTaskManager;
+    QgsFieldFormatterRegistry* mFieldFormatterRegistry;
```

Comment:
I think explicit deleting here, where we can control the order of deletion, is actually safer. It's likely in future we'll have members here which are dependent on others and need to be carefully deleted one by one.